### ISSUE TITLE
USDA_FNS_Reducing Churn for SNAP Recipients (1719)

### ISSUE THREAD

**Issue by jwbowers** [gsa-oes/researchsupportsquad#13]

**Comment by jwbowers** [comment]
@michaelhand will work on this.

**Comment by michaelhand** [comment]
@jwbowers: I was looking through project documentation and tracker last night; is this really ready for re-analysis? Seems like it just started Lead Analysis. Maybe I'm missing something; we can bring this up on the call if necessary, but thought I'd start the thread here in case we can resolve offline.

**Comment by jwbowers** [comment]
@michaelhand It seemed like something that Kelly had on her list of up next. Let's ping her by email.

**Comment by michaelhand** [comment]
@jwbowers A related question: for projects with limited or no documentation or data in the project folder, is it SOP to just coordinate with the project lead to get the design details and data?

**Comment by jwbowers** [comment]
Right.  Probably good to have those things in a spot on Google Drive once you get them from the Lead, too.
